src/main/kotlin/uk/gov/dluhc/printapi/service/S3PhotoService.kt

```diff
@@ -0,0 +1,29 @@
+package uk.gov.dluhc.printapi.service
+
+import mu.KotlinLogging
+import org.springframework.stereotype.Service
+import software.amazon.awssdk.core.exception.SdkException
+import software.amazon.awssdk.services.s3.S3Client
+import software.amazon.awssdk.services.s3.model.DeleteObjectRequest
+
+private val logger = KotlinLogging.logger {}
+
+@Service
+class S3PhotoService(
+    private val s3Client: S3Client
+) {
+    /**
+     * Removes the photo that is on the printed "Elector Document" (i.e. Certificate/AED) from S3.
+     */
+    fun removePhoto(photoS3Arn: String) {
```

Comment:
Could this be generic method `removeObjectFromS3(s3Arn: String)` - Not a blocker

Reply:
I considered that but thought we'll only ever get/remove photos within S3 in the print-api. By making it generic I think you lose some of the readability.